db/migrate/20240221113542_create_statements.rb

```diff
@@ -0,0 +1,21 @@
+class CreateStatements < ActiveRecord::Migration[7.1]
+  def change
+    create_enum :statement_states, %w[open payable paid]
+
+    create_table :statements do |t|
+      t.integer :month, null: false
+      t.integer :year, null: false
+      t.date :deadline_date
+      t.date :payment_date
+      t.boolean :output_fee, default: true, null: false
+      t.references :cohort, null: false, foreign_key: true
+      t.references :lead_provider, null: false, foreign_key: true
+      t.datetime :marked_as_paid_at
```

Comment:
Another question to the uninitiated, what's the difference between `payment_date` and `marked_as_paid_at` (other than it being a `datetime`)

Reply:
`marked_as_paid_at` when we action statement to be paid, it is used to display some text and for background job:
https://dfedigital.atlassian.net/browse/CPDLP-2295

Reply:
Ah ok, so:

* `payment_date` is the date on which the payment _was_ made
* `marked_as_paid_at` is the date and time *we recorded it* as being paid